src/components/Review.jsx

```diff
@@ -0,0 +1,183 @@
+                                            }
+                                        >
+                                            <img
+                                                src={
+                                                    reviewData.rating >= index
+                                                        ? "../../public/star.png"
+                                                        : "../../public/star-empty.png"
+                                                }
+                                            />
+                                        </button>
+                                    </li>
+                                );
+                            })}
+                        </ol>
+                    </section>
+
+                    <section className="input-section">
+                        <label htmlFor="comment">Komentarz</label>
+                        <input
+                            id="comment"
+                            type="comment"
+                            name="comment"
+                            onChange={(event) => inputHandler("comment", event)}
+                            value={reviewData.comment}
+                        />
+                    </section>
+
+                    <section className="input-section tip">
+                        <label htmlFor="tip">Napiwek</label>
+                        <div className="tip-buttons-grid">
+                            <button
+                                type="button"
+                                onClick={() => tipButtonHandler(0)}
+                            >
+                                0
+                            </button>
+                            <button
+                                type="button"
+                                onClick={() => tipButtonHandler(5)}
+                            >
+                                5
+                            </button>
+                            <button
+                                type="button"
+                                onClick={() => tipButtonHandler(10)}
+                            >
+                                10
+                            </button>
+                            <button
+                                type="button"
+                                onClick={() => tipButtonHandler(20)}
+                            >
+                                20
+                            </button>
+                            <input
+                                id="tip"
+                                type="number"
+                                name="tip"
+                                onChange={(event) =>
+                                    inputHandler("tipAmount", event)
+                                }
+                                value={reviewData.tipAmount}
+                            />
+                        </div>
```

Comment:
1. Looks like a separate component
2. Could be done with map (like stars above)
example:
const tips = [0,5,10,20]

tips.map(tip=>...)

Reply:
fixed